src/os/bluestore/BlueStore.cc

```diff
@@ -8920,60 +8823,59 @@ int BlueStore::_do_read(
     while (b_len > 0) {
       unsigned l;
       if (pc != cache_res.end() &&
-	  pc->first == b_off) {
-	l = pc->second.length();
-	ready_regions[pos].claim(pc->second);
-	dout(30) << __func__ << "    use cache 0x" << std::hex << pos << ": 0x"
-		 << b_off << "~" << l << std::dec << dendl;
-	++pc;
+          pc->first == b_off) {
+        l = pc->second.length();
+        ready_regions[pos].claim(pc->second);
+        dout(30) << __func__ << "    use cache 0x" << std::hex << pos << ": 0x"
+                 << b_off << "~" << l << std::dec << dendl;
+        ++pc;
       } else {
-	l = b_len;
-	if (pc != cache_res.end()) {
-	  ceph_assert(pc->first > b_off);
-	  l = pc->first - b_off;
-	}
-	dout(30) << __func__ << "    will read 0x" << std::hex << pos << ": 0x"
-		 << b_off << "~" << l << std::dec << dendl;
-	// merge regions
-	{
-	  uint64_t r_off = b_off;
-	  uint64_t r_len = l;
-	  uint64_t front = r_off % chunk_size;
-	  if (front) {
-	    r_off -= front;
-	    r_len += front;
-	  }
-	  unsigned tail = r_len % chunk_size;
-	  if (tail) {
-	    r_len += chunk_size - tail;
-	  }
-	  bool merged = false;
-	  regions2read_t& r2r = blobs2read[bptr];
-	  if (r2r.size()) {
-	    read_req_t& pre = r2r.back();
-	    if (r_off <= (pre.r_off + pre.r_len)) {
-	      front += (r_off - pre.r_off);
-	      pre.r_len += (r_off + r_len - pre.r_off - pre.r_len);
-	      pre.regs.emplace_back(region_t(pos, b_off, l, front));
-	      merged = true;
-	    }
-	  } 
-	  if (!merged) {
-	    read_req_t req(r_off, r_len);
-	    req.regs.emplace_back(region_t(pos, b_off, l, front));
-	    r2r.emplace_back(std::move(req));
-	  }
-	}
-	++num_regions;
+        l = b_len;
+        if (pc != cache_res.end()) {
+          ceph_assert(pc->first > b_off);
+          l = pc->first - b_off;
+        }
+        dout(30) << __func__ << "    will read 0x" << std::hex << pos << ": 0x"
+                 << b_off << "~" << l << std::dec << dendl;
+        // merge regions
+        {
+          uint64_t r_off = b_off;
+          uint64_t r_len = l;
+          uint64_t front = r_off % chunk_size;
+          if (front) {
+            r_off -= front;
+            r_len += front;
+          }
+          unsigned tail = r_len % chunk_size;
+          if (tail) {
+            r_len += chunk_size - tail;
+          }
+          bool merged = false;
+          regions2read_t& r2r = blobs2read[bptr];
+          if (r2r.size()) {
+            read_req_t& pre = r2r.back();
+            if (r_off <= (pre.r_off + pre.r_len)) {
+              front += (r_off - pre.r_off);
+              pre.r_len += (r_off + r_len - pre.r_off - pre.r_len);
+              pre.regs.emplace_back(region_t(pos, b_off, l, front));
+              merged = true;
+            }
+          }
+          if (!merged) {
+            read_req_t req(r_off, r_len);
+            req.regs.emplace_back(region_t(pos, b_off, l, front));
+            r2r.emplace_back(std::move(req));
+          }
+        }
```

Comment:
@xiexingguo i'd suggest avoid massively reformatting source code in future.

Reply:
sorry, will pay attention to next time